cl/alerts/management/commands/cl_send_alerts.py

```diff
@@ -147,7 +154,11 @@ def run_query(self, alert, rate):
                 # Bail out. No results will be found if no valid_ids.
                 return query_type, results
 
-            main_params = search_utils.build_main_query(cd, facet=False)
+            main_params = search_utils.build_main_query(
+                cd,
+                highlight="text",  # Required to show all field as in Search API
```

Comment:
This change was also required to show all the fields as in the search API, if we omit this all the fields are highlighted and they're not serialized, so I changed keys in the email templates according to this change.